test/SlowTests/Issues/RavenDB_15430.cs

```diff
@@ -173,6 +174,17 @@ record = await store.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation
             }
         }
 
+        private void Shuffle(List<string> list, Random random)
+        {
+            int n = list.Count;
+            while (n > 1)
+            {
+                n--;
+                int k = random.Next(n + 1);
+                (list[k], list[n]) = (list[n], list[k]);
+            }
+        }
```

Comment:
we already have this under `ThreadSafeRandom`

Reply:
done - https://github.com/ravendb/ravendb/pull/15162